app/assets/javascripts/discourse/lib/plugin-api.js.es6

```diff
@@ -753,6 +754,21 @@ class PluginApi {
     Sharing.addSharingId(options.id);
     Sharing.addSource(options);
   }
+
+  /**
+   *
+   * Registers a function to handle uploads for specified file types
+   * The normal uploading functionality will be bypassed
+   *
+   * Example:
+   *
+   * addComposerUploadHandler(["mp4", "mov"], (file) => {
+   *    console.log("Handling upload for", file.name);
+   * })
+   */
+  addComposerUploadHandler(extensions, method) {
+    addComposerUploadHandler1(extensions, method);
```

Comment:
The `1` here is not needed, I think it just makes stuff a bit more confusing cause no other method follows that pattern